src/Microsoft.ML.StandardLearners/Standard/MultiClass/PkpdTrainer.cs

```diff
@@ -42,17 +42,17 @@ namespace Microsoft.ML.Trainers
     /// pair.
     ///
     /// These two can allow you to exploit trainers that do not naturally have a
-    /// multiclass option, for example, using the Runtime.FastTree.FastTreeBinaryClassificationTrainer
+    /// multiclass option, for example, using the Microsoft.ML.Trainers.FastTree.FastTreeBinaryClassificationTrainer
     /// to solve a multiclass problem.
     /// Alternately, it can allow ML.NET to solve a "simpler" problem even in the cases
     /// where the trainer has a multiclass option, but using it directly is not
-    /// practical due to, usually, memory constraints.For example, while a multiclass
+    /// practical due to, usually, memory constraints. For example, while a multiclass
     /// logistic regression is a more principled way to solve a multiclass problem, it
     /// requires that the learner store a lot more intermediate state in the form of
     /// L-BFGS history for all classes *simultaneously*, rather than just one-by-one
     /// as would be needed for OVA.
```

Comment:
nit: If we are renaming OVA to OneVsAll, I'd also change it here.

Reply:
ah! Thanks, I will update.

---
In reply to: [260539673](https://github.com/dotnet/machinelearning/pull/2749#discussion_r260539673) [](ancestors = 260539673)